lib/compiler.js

```diff
@@ -65,6 +65,13 @@ exports.compile = function (code, options) {
     });
 
     importExportVisitor.finalizeHoisting();
+
+    if (getOption(options, "wrapWithRun")) {
+      result.code = "module.run(function(){" +
+        magicString.toString() + "//*/\n});";
+
+      return result;
+    }
```

Comment:
I'm pretty sure running all REPL code inside a function wrapper will cause more problems than it solves, and lead to unpleasant surprises.

For example, if I flip `wrapWithRun` to `true` in the REPL, I can't use any of the symbols I've imported, because the variable declaration ends up inside the `module.run` callback. Hoisting those variables outside the `module.run` callback feels like it would be a very big departure from the way things currently work (where the variable declaration is always right before the import).